src/app/app.module.ts

```diff
@@ -3,16 +3,16 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './components/header/header.component';
+import { FooterComponent } from './components/footer/footer.component';
+import { LogoComponent } from './components/logo/logo.component';
+import { CourseItemComponent } from './components/course-item/course-item.component';
+import { CourseListComponent } from './components/course-list/course-list.component';
```

Comment:
Please, create in the components folder file index.ts, and export each component from there like export * from 'path to component".   It gives you benefit in organizing imports and avoiding multiple import.